addons/im_livechat/static/tests/channel_invite.test.js

```diff
@@ -124,3 +124,45 @@ test("Partners invited most frequently by the current user come first", async ()
     await contains(":nth-child(1 of .o-discuss-ChannelInvitation-selectable)", { text: "John" });
     await contains(":nth-child(2 of .o-discuss-ChannelInvitation-selectable)", { text: "Albert" });
 });
+
+test("Indicate operators are already in a call", async () => {
```

Comment:
Would rename to "shows operators are in call"
- already is too opinionated in thinking invite is necessarily to invite in another call, so drop
- "indicate" is more about something waving or pointing fingers, which is not what the UI shows. Prefer generic "show" in this case.